app/dashboard/models.py

```diff
@@ -397,6 +397,7 @@ class Bounty(SuperModel):
         default=False, help_text=_('This bounty will be part of the hypercharged bounties')
     )
     hyper_next_publication = models.DateTimeField(null=True, blank=True)
+    is_pro = models.BooleanField(help_text=_('Is this use upgraded to pro?'))
```

Comment:
*user

Reply:
I think this need to be on profile not on the bounty model as context is looking for `profile.is_pro` Unless Im missing something.

Reply:
oh shoot ur right

Reply:
coding is hard, thats why im a powerpoint guy these days.

fixed at ba14183f3f0016013058cbfc136ec505df2b9023